source/kraken/maintenance_worker.cpp

```diff
@@ -202,6 +202,7 @@ void MaintenanceWorker::handle_rt_in_batch(const std::vector<AmqpClient::Envelop
         }
     }
     if (data) {
+        data->pt_data->build_autocomplete(*(data->geo_ref));
```

Comment:
So, when you load a new realtime file you rebuild autocomplete. 
I don't know if it is necessary to do this if the new GTF-RT if different to a ADD NEWE TRIP, no ?

Reply:
You are right it's useful only when adding/removing ptref objects (so currently for a trip-add)

Yep, it can be challenged, here are my thinking. Any opinion welcome.
I chose this solution because:
* it's done only at the end of a batch (mutualizing things as much as possible)
* it doesn't seem very expensive now
* it requires having a `mut data` and it's not the case below (when knowing if trip is 'added' or not)

An even better (in perf) solution would be to flag pt_data that something was changed and trigger this only when it's the case.
But this seems a bit complicated and difficult to maintain given the cost.

Maybe I can block it depending on the param enabling trip-add. Not sure it's super usefull (perf or safety) and maybe it can mess later features (other params to handle?).